smartsim/_core/control/manifest.py

```diff
@@ -82,13 +84,13 @@ def models(self):
         return _models
 
     @property
-    def ensembles(self):
+    def ensembles(self) -> t.List[EntityList]:
         """Return Ensemble instances in Manifest
 
         :return: list of ensembles
         :rtype: List[Ensemble]
         """
-        _ensembles = []
+        _ensembles: t.List[EntityList] = []
```

Comment:
You seem to have declared type on some, but not all, local variables. What's the defining point for when it's needed?

Reply:
mypy identifies the items that can't be inferred. I err'd on the side of less clutter rather than adding a hints on variables where it wasn't necessary